modules/irsa/variables.tf

```diff
@@ -65,3 +59,18 @@ variable "irsa_iam_permissions_boundary" {
   default     = ""
   description = "IAM Policy ARN for IRSA IAM role permissions boundary"
 }
+
+variable "addon_context" {
+  type = object({
+    aws_caller_identity_account_id = string
+    aws_caller_identity_arn        = string
+    aws_eks_cluster_endpoint       = string
+    aws_partition_partition        = string
+    aws_region_id                  = string
+    aws_region_name                = string
+    eks_oidc_issuer_url            = string
+    eks_oidc_provider_arn          = string
+    tags                           = map(string)
```

Comment:
What is `tags` in the context since its not a data source lookup? Should the `eks_cluster_id` be in the context object too in that case?

Reply:
agree

Reply:
Yes, the `eks_cluster_id` is must.